src/main/java/com/serenitydojo/Dog.java

```diff
@@ -0,0 +1,42 @@
+package com.serenitydojo;
+
+public class Dog {
+
+    private String name;
+    private String favoriteToy;
+    private int age;
+
+    private String makeNoise;
+    private boolean isFed;
+
+
+    public static final String DOG_NOISE = "Woof";
+    public Dog(String name, String favoriteToy, int age, String makeNoise, boolean isFed) {
+        this.name = name;
+        this.favoriteToy = favoriteToy;
+        this.age = age;
+        this.makeNoise = makeNoise;
+        this.isFed = true;
+    }
+
+
+    public String getName() {
+        return name;
+    }
+
+    public String getFavoriteToy() {
+        return favoriteToy;
+    }
+
+    public int getAge() {
+        return age;
+    }
+
+    public String makeNoise(){
+        return DOG_NOISE;
+    }
+
+    public boolean feed(){
```

Comment:
this confuses me. everything is "getX" so shouldnt this also be like
```suggestion
    public boolean isFed(){
```
`feed()` suggests that by calling this method you will feed the dog, but it's actually just a getter

Reply:
Okay, so I actually got this from exercise 4, they asked I create a "feed" method  - https://screenrec.com/share/YpEedV10g5  and then I set it to true here - https://screenrec.com/share/b0CmPD67Yk 

Please Is it correct?

Reply:
ah, I see.

what's meant is that you create a feed method
```
void feed() {
  isFed = true;
}
```

don't change the constructor, it should be `this.isFed = isFed;` otherwise it would always be fed=true, even if you pass "false" as parameter

Reply:
Yes, I just made the corrections. Kindly recheck and let me know if I could merge.

Reply:
sure, the rest looks allright to me

Reply:
So happy. Thanks